WordPress/src/main/java/org/wordpress/android/ui/media/services/MediaUploadService.java

```diff
@@ -272,4 +289,24 @@ public void onMediaUploaded(OnMediaUploaded event) {
             handleOnMediaUploadedSuccess(event);
         }
     }
+
+    /**
+     * Analytics about media being uploaded
+     *
+     * @param media The media being uploaded
+     */
+    private void trackUploadMediaEvents(AnalyticsTracker.Stat stat, MediaModel media) {
+        if (media == null) {
+            AppLog.e(AppLog.T.MEDIA, "Cannot track media upload service events if the original media is null!!");
+            return;
+        }
+
+        if (availableAnalitycs != null && availableAnalitycs.contains(stat)) {
```

Comment:
I'm confused about why we're bothering to check the available analytics here, since this routine is only called with the four stats that belong to the set.